CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
+
+All notable changes to this project will be documented in this file. 
```

Comment:
changelog and histroy md should match and also mention as breaking change that support for 3.6 <  is dropped